app/common/lib/ledgerUtil.js

```diff
@@ -524,7 +555,10 @@ const getMediaProvider = (url, firstPartyUrl, referrer) => {
       (firstPartyUrl && firstPartyUrl.startsWith('https://www.twitch.tv/')) ||
       (referrer && referrer.startsWith('https://player.twitch.tv/'))
     ) &&
-    url.startsWith('https://api.mixpanel.com/')
+    (
+      url.includes('.ttvnw.net/v1/segment/') ||
+      url.includes('https://ttvnw.net/v1/segment/')
```

Comment:
Reading this if can be a little difficult; one approach would be to pull the conditions into variables. Something like:
```
const firstPartyTwitch = firstPartyUrl && firstPartyUrl.startsWith('https://www.twitch.tv/')
const referrerTwitch = referrer && referrer.startsWith('https://player.twitch.tv/')
const isTwitch = (firstPartyTwitch || referrerTwitch) && [
  '.ttvnw.net/v1/segment/',
  'https://ttvnw.net/v1/segment/'
].some(segment => segment.includes(url))

if (isTwitch) {
  return ledgerMediaProviders.TWITCH
}

```